aiobotocore/paginate.py

```diff
@@ -59,12 +59,14 @@ def _init_pager(self):
         self._starting_truncation = 0
         self._inject_starting_params(self._current_kwargs)
 
-    @asyncio.coroutine
-    def next_page(self):
+    async def __aiter__(self):
+        return self
+
+    async def __anext__(self):
```

Comment:
I did it like this because I saw your comment about wanting to get rid of python <3.5.  I can make it backwards compatible as well.  Up to you :)